pkg/backend/backend.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 const (
-	maxWorkers      = 100
+	maxWorkers      = 0
```

Comment:
Probably should drop this commit now.  :slightly_smiling_face:

Reply:
dropped!